docs/pages/Workspace.md

```diff
@@ -0,0 +1,125 @@
+
+The `name` property must satisfy the following conditions. They are identical to [project names](./Configuration.md#name):
+
+* Must be at least 3 characters long
+* Must be no longer than 50 characters
+* Must contain lowercase characters only
+* Must contain alphanumeric characters, dash, underscore and period only
+    - Exception: The `@` and `/` characters are allowed at certain positions as
+      explained below
+* Must start with an alphabetic character or an `@` character
+* If it starts with an `@` character, it must contain exactly one
+  forward slash `/`
+    - This is aligned with the npm concept for package scopes, for example `@org/lib.name`
+
+## Dependency Management
+
+UI5 Workspace configurations allow to influence the dependency resolution when working with a UI5 project.
+
+!!! example
+    ```yaml
+    specVersion: workspace/1.0
+    metadata:
+        name: dolphin
+	dependencyManagement:
+   		resolutions:
+       		- path: ../light.library
+       		- path: ../heavy.library
+       		- path: ../test.library
+    ```
+
+### Resolutions
+
+```yaml
+dependencyManagement:
+   	resolutions:
+       	- path: ../test.library
+```
+
+Resolution paths will be used by UI5 Tooling to look for project dependencies. Any dependencies found via these paths will be used over the ones found with the regular dependency resolution mechanism.
+
+Paths must point to a directory, which must contain a `package.json`. UI5 Tooling will attempt to find a UI5 Project here. However, if the `package.json` contains a |`workspaces`](https://docs.npmjs.com/cli/v8/using-npm/workspaces) or equivalent `ui5.workspaces` configuration (which overrules the first), UI5 Tooling will resolve the workspace first and attempt to find UI5 Projects in the configured "npm workspaces". This is commonly used in monorepos. An example can be found in the [OpenUI5 repository](https://github.com/SAP/openui5/blob/b4267488e5d3546de4cd9577ccac4208482d71e0/package.json#L130-L132).
+
+Paths must be written in POSIX (i.e. using only forward slashes `/` as path segment separators) and must be **relative to the workspace configuration file**. Absolute paths, or paths relative to the home directory (`~`) are not allowed. This is to ensure that workspace configuration files are easy to share and to reduce the chance of them exposing sensitive information like usernames or large directory hierarchies. Symbolic links are followed.
+
+Note that this configuration only affects the resolution of dependencies which have been already found during the regular dependency resolution process of a project. For example, if a workspace resolution path resolves to a project that would otherwise not be part of the dependency tree of the current root project, it will not be added to the dependency tree. Also, transitive dependencies of resolved projects are not being followed.
```

Comment:
```suggestion
Note that this configuration only affects the resolution of dependencies which have already been found during the regular dependency resolution process of a project. For example, if a workspace resolution path resolves to a project that would otherwise not be part of the dependency tree of the current root project, it will not be added to the dependency tree. Also, transitive dependencies of resolved projects are not being followed.
```